composer.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "yiisoft/cache-db",
+    "type": "library",
+    "description": "Yii Caching Library - DB Handler",
+    "keywords": [
+        "yii",
+        "framework",
+        "cache",
+        "psr-16"
+    ],
+    "homepage": "http://www.yiiframework.com/",
+    "license": "BSD-3-Clause",
+    "support": {
+        "source": "https://github.com/yiisoft/cache-db",
+        "issues": "https://github.com/yiisoft/cache-db/issues",
+        "forum": "http://www.yiiframework.com/forum/",
+        "wiki": "http://www.yiiframework.com/wiki/",
+        "irc": "irc://irc.freenode.net/yii"
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": "^7.4|^8.0",
+        "psr/simple-cache": "~1.0.1",
+        "yiisoft/db": "^3.0@dev",
+        "yiisoft/yii-db-migration": "^1.0@dev"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.4",
+        "roave/infection-static-analysis-plugin": "^1.6",
+        "spatie/phpunit-watcher": "^1.23",
+        "vimeo/psalm": "^4.4",
+        "yiisoft/db-sqlite": "^3.0@dev",
+        "yiisoft/di": "^3.0@dev"
+    },
+    "provide": {
+        "psr/simple-cache-implementation": "1.0.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "Yiisoft\\Cache\\Db\\": "src"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Yiisoft\\Cache\\Db\\Tests\\": "tests"
+        }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "3.0.x-dev"
+        },
+        "config-plugin": {
+            "common": "config/common.php",
```

Comment:
Either this file common.php is missing, or this line should be removed.

Reply:
Removed.